### DISCUSSION TITLE
TRPG-OCL-Rule について OCL

### DISCUSSION THREAD

**Discussion by orga-itsuka-trpg** [orga-itsuka-trpg/TRPG-OCL-Rule#223] [Announcements]
## 👋 ディスカッション
ここでは下記の内容について質問・回答・議論できることを教える
  * このリポジトリについて質問
  * Git/GitHubの使い方やその他ログやファイルの管理について質問
  * 何らかのアイデアの共有

このリポジトリの内容やGitの使い方について質問したくなった時、スレで聞くのも何だけど `orga-itsuka-trpg` の確認を待つのも面倒だな…と思った時はここを使用してくれ(^^)

**Comment by ShanShan1129** [comment]
来たか　話し合いできるところ
あと少しでログ整理用卓とか立てるところだったのでありがたいだろ

**Comment by orga-itsuka-trpg** [comment]
ここは一応アナウンス用のトピックとしてピン留めとコメント等追加でできないようロックすることを教える
GitおよびGitHubについて操作がわからなくなった際に参考になりそうな資料だ　飲んでくれ
[サル先生のGit入門](https://backlog.com/ja/git-tutorial/)
[Git / Githubを使ったチーム開発](https://zenn.dev/penguin4731/books/11bee5738e367f)
[GitHub Skills](https://skills.github.com/)
